### PR TITLE
Add eth_blobGasPrice/eth_gasPrices; update eth_feeHistory

### PR DESCRIPTION
**Pull request by flcl42**
Implements https://github.com/ethereum/execution-apis/pull/486

## Changes

- Add eth_blobBaseFee
- Update eth_feeHistory with blob fields

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [x] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [x] Yes
- [ ] No

#### If yes, did you write tests?

- [x] Yes
- [ ] No

### CODE REVIEW

File: src/Nethermind/Nethermind.JsonRpc/Modules/Eth/FeeHistory/FeeHistoryOracle.cs [hunk 1]
```diff
@@ -66,7 +75,7 @@ public ResultWrapper<FeeHistoryResults> GetFeeHistory(long blockCount, BlockPara
                 block = _blockFinder.FindParent(block, BlockTreeLookupOptions.RequireCanonical);
             }
 
-            FeeHistoryResults feeHistoryResults = new(oldestBlockNumber, baseFeePerGas.ToArray(), gasUsedRatio.ToArray(), rewards?.ToArray());
+            FeeHistoryResults feeHistoryResults = new(oldestBlockNumber, baseFeePerGas.ToArray(), gasUsedRatio.ToArray(), baseFeePerBlobGas.ToArray(), blobGasUsedRatio.ToArray(), rewards?.ToArray());
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.JsonRpc/Modules/Eth/FeeHistory/FeeHistoryOracle.cs:78]: Can we make FeeHistoryResult disposable, and change all collections to ArrayPoolList and dispose it there? We wouldn't have to allocate almost anything.

**Review by LukaszRozmej — APPROVED** [review]

**Review by marcindsobczak — APPROVED** [review]

File: src/Nethermind/Nethermind.JsonRpc/Modules/Eth/FeeHistory/FeeHistoryOracle.cs (hunk 1 above)
Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.JsonRpc/Modules/Eth/FeeHistory/FeeHistoryOracle.cs:78]: lets do it in next PR

**Review by benaadams — APPROVED** [review]
Any hive tests for this?

**Comment by flcl42** [comment]
> Any hive tests for this?

Not yet

**Pull request merged**